routers/web/repo/issue.go

```diff
@@ -136,6 +138,28 @@ func MustAllowPulls(ctx *context.Context) {
 	}
 }
 
+func dependenciesToHTML(ctx *context.Context, dependencies []*issues_model.DependencyInfo) template.HTML {
+	if len(dependencies) == 0 {
+		return ""
+	}
+
+	htmlCode := "<span>"
+
+	for index, dependency := range dependencies {
+		if index != 0 {
+			htmlCode += `<span>,</span>`
+		}
+		anchorClasses := "gt-ml-2"
+		if dependency.Issue.IsClosed {
+			anchorClasses += " gt-line-through"
+		}
+		htmlCode += fmt.Sprintf(`<a href="%s" data-tooltip-content="#%d %s" class="%s">#%d</a>`,
```

Comment:
I think you can add a `ref-issue` class to the link in place of `data-tooltip-content`. It should render a popup preview on hover.